javascript/ql/src/semmle/javascript/frameworks/Vue.qll

```diff
@@ -0,0 +1,361 @@
+/**
+ * Provides classes for working with Vue code.
+ */
+
+import javascript
+import semmle.javascript.security.dataflow.DomBasedXss
```

Comment:
I think importing transitively this from `javascript.qll` will cause performance issues, since every dataflow query must then also evaluate `DomBasedXss`.

Reply:
Fixed.